### PR TITLE
Document help wanted issue queries

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/process/community-guidelines.rst [hunk 1]
```diff
@@ -157,6 +157,21 @@ Once a pull request for an open GitHub Help Wanted issue is merged, please ensur
 - the corresponding GitHub Help Wanted issue is closed, thanking the contributor
 - the corresponding JIRA ticket is resolved
 
+Help Wanted Issue Links
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Below is a list of permanent redirects to Help Wanted issue queries:
+
+1. `mattermost/mattermost-server` - https://mattermost.com/pl/help-wanted-mattermost-server which redirects to https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Help+Wanted%22+label%3A%22Tech%2FGo%22
```
Comment (hanzei) [inline source/process/community-guidelines.rst]: One last nit: Can you please also add `is:issue` to this, 3., 4. and 5.? Although there should never be a PR with this labels, it's better safe to be sorry.

Comment (jasonblais) [reply source/process/community-guidelines.rst]: Done 👍

**Comment by amyblais** [comment]
@jasonblais Do we merge this too?

**Comment by jasonblais** [comment]
Yes, good catch, merging

**Pull request merged**